include/item.h

```diff
@@ -14,7 +14,7 @@ struct Item
     u8 holdEffectParam;
     const u8 *description;
     u8 importance;
-    u8 unk19;
+    u8 registrability;
```

Comment:
There should probably be a comment here indicating that this field has no effect so that the name doesn't mislead people. Also across all repos it's only ever 0/1, so it should be a `bool8` and use `TRUE`.